docs/NPC_Swap_Tutorial.md

```diff
@@ -0,0 +1,224 @@
+---
+sidebar_position: 1
+description: An introductory modding guide that will show you how to swap 47's Signature Suit with an NPC.
+---
+
+# Recommended reading 
+* [Glacier 2 Modding Basics](https://wiki.notex.app/glacier2/modding_basics)
+* [Chunk Data](https://wiki.notex.app/glacier2/chunkdata)
+
+# Requirements
+* [RPKG Tool](https://notex.app/rpkg/)
+* [Simple Mod Framework](https://www.nexusmods.com/hitman3/mods/200)
+* A program to read and edit JSON files (Notepad++ is recommended, but any text editor will suffice in this scenario as we're only changing a single value at the beginning of the document. Otherwise, QuickEntity Editor is the superior choice when editing QN JSON files)

```

Comment:
I would personally recommend VSCode over NP++ (and I'm sure many others will too, but to each their own).

Reply:
I would suggest VSCodium, but Tony is correct, I would avoid specifically recommending Notepad++ and instead suggest just a basic code editor and allow the user to do the rest.

Reply:
Sounds good, I've swapped Notepad++ for VSCode.